bay/images/Dockerfile.ci-builder

```
@@ -1,40 +1,64 @@
 FROM hashicorp/terraform:light AS terraform
-FROM integratedexperts/ci-builder:latest
+FROM uselagoon/php-7.4-cli:latest
+
```

Maybe make /tmp a volume 

```suggestion
VOLUME /tmp
```